tests/real/community.rs

```diff
@@ -88,12 +88,24 @@ async fn server_state() {
 
     // examine server state before pushing anything
     let server_state = client.current_info().await.unwrap();
+    // the Faktory release we are writing bindings and testing
+    // against is at least "1.8.0"
+    assert_gte!(server_state.server.version.major, 1);
```

Comment:
I would probably make this `assert_eq!`, since if we get a bump to `2`, the `>=` on `minor` would no longer work right.